src/fr/mrc/ptichat/main/utils/MessageSignatureController.java

```diff
@@ -9,7 +9,7 @@
     public MessageSignatureController() {
         ps = new Properties();
         try {
-            ps.load(new java.io.FileInputStream("./src/resources/messageSignature.properties"));
+            ps.load(new java.io.FileInputStream("./src/fr/mrc/ptichat/resources/messageSignature.properties"));
```

Comment:
As you said, hard coded path is bad. I will fix this in next commit using a ResourceBundle instead of the Properties class :)